deploy/autopg.Dockerfile

```
@@ -0,0 +1,96 @@
+FROM --platform=$BUILDPLATFORM debian:bookworm-slim AS fetch
+ARG AUTOPG_VERSION
+ARG PG_RUNTIME_VERSION
+ARG TARGETARCH
+
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends ca-certificates curl \
+  && rm -rf /var/lib/apt/lists/*
+
+# Map TARGETARCH → release-asset suffix + autopg runtime platform key.
+# amd64 uses the glibc build (final base is debian bookworm).
+RUN set -eu; \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)